index.html

```diff
@@ -273,31 +273,31 @@ <h2>On behalf of organisations</h2>
 
 				<p>Os motores de busca estão numa posição poderosa para exercer influência que resolva esse problema. No entanto, o Google escolheu criar uma posição privilegiada na parte superior de seus resultados de busca (para artigos) e um ícone de "relâmpago" (para todos os tipos de conteúdo), que só são acessíveis para editores que usam a tecnologia controlada pelo Google, atendidos pelo mesmo a partir de suas infraestruturas, em um URL do Google e colocado dentro de uma experiência de usuário controlada pelo próprio Google. (<a href='https://developers.google.com/search/docs/guides/about-amp'>fonte</a>)</p>
 
-				<p>O formato AMP em si não é um problema, mas dois aspectos de sua implementação reforçam a posição do Google como uma plataforma padrão para conteúdos, já que o Google busca impulsionar a incorporação de AMP com criadores de conteúdo:</p>
+				<p>O formato AMP em si não é um problema, mas dois aspectos de sua implementação reforçam a posição do Google como uma plataforma padrão para conteúdos, já que o Google busca impulsionar a incorporação do AMP por criadores de conteúdo:</p>
 
 				<ol>
 					<li>O conteúdo que "optar pelo" AMP e a hospedagem associada ao domínio do Google é concedido a promoção de pesquisa preferencial, incluindo, para artigos de notícias, uma posição acima de todos os outros resultados.</li>
 					<li>Quando um usuário navega do Google para um conteúdo que o Google recomendou, eles ficam inconscientemente dentro do ecossistema do Google.</li>
 				</ol>
 
-				<p>Se o objetivo do Google com a AMP é realmente melhorar a experiência do usuário na Web, sugerimos algumas mudanças simples, permitindo que a Web permaneça dinâmica, competitiva e orientada para o consumidor:</p>
+				<p>Se o objetivo do Google com o AMP é realmente melhorar a experiência do usuário na Web, sugerimos algumas mudanças simples, permitindo que a Web permaneça dinâmica, competitiva e orientada para o consumidor:</p>
 
 				<ol>
-					<li>Em vez de conceder colocação privilegiada nos resultados de pesquisa apenas para a AMP, forneça as mesmas vantagens a todas as páginas que atendam a um critério de desempenho objetivo e neutro, como o Índice de Velocidade. Os editores podem usar qualquer solução técnica de sua escolha.</li>
+					<li>Em vez de conceder colocação privilegiada nos resultados de pesquisa apenas para o AMP, forneça as mesmas vantagens a todas as páginas que atendam a um critério de desempenho objetivo e neutro, como o Índice de Velocidade. Os editores podem usar qualquer solução técnica de sua escolha.</li>
```

Comment:
Just a thing I missed: "Índice de Velocidade" should retain its original name, Speed Index, as it doesn't have a Portuguese localization